.ado/templates/run-windows-with-certificates.yml

```diff
@@ -23,20 +23,32 @@ parameters:
     type: string
 
 steps:
-  - task: Cache@2
-    inputs:
-      key: '"run-windows-with-certificates" | "${{ parameters.workingDirectory }}" | ${{ parameters.workingDirectory }}/windows/NuGet.config | **/packages.config | **/packages.lock.json'
-      restoreKeys: '"run-windows-with-certificates" | "${{ parameters.workingDirectory }}" | ${{ parameters.workingDirectory }}/windows/NuGet.config'
-      path: ${{ parameters.workingDirectory }}/windows/packages/
-    displayName: Copy cached NuGet packages
+  - template: ../templates/cache-nuget-restore.yml
+    parameters:
+      solutionPath: ${{ parameters.workingDirectory }}
+      nugetConfigPath: ${{ parameters.workingDirectory }}/windows/NuGet.config
+      repositoryPath : ${{ parameters.workingDirectory }}/windows/packages
 
   - ${{ if eq(parameters.buildConfiguration, 'Debug') }}:
-    - script: yarn windows --no-packager --no-launch ${{ parameters.deployOption }} --arch ${{ parameters.buildPlatform }} --logging --buildLogDirectory ${{ parameters.buildLogDirectory }}
+    - script: >
+        yarn windows
+        --arch ${{ parameters.buildPlatform }}
```

Comment:
(nit)

Indent arguments one level beyond command?
i.e.:

```
  yarn windows
    --arch ...
```

Reply:
I was seeing some issues with that, where I needed to deindent :/